Header header
nav_msgs/Odometry car_state
std_msgs/Bool isMaskDetected
std_msgs/Bool isCarNinety
geometry_msgs/Vector3 car_centre
geometry_msgs/Vector3 car_back
geometry_msgs/Vector3 car_front
